guides/development/react-guidelines.md

```diff
@@ -72,7 +74,50 @@ We currently use:
 - REST: [react-query](https://github.com/tannerlinsley/react-query)
 - GraphQL: [apollo-client](https://github.com/apollographql/apollo-client)
 
-## 2. <a name='Generalprojectorganizationandarchitecture'></a>General project organization and architecture
+###  1.6. <a name='Dousefunctiondeclarations'></a>Do use function declarations
+
+For a better readability.
+
+```ts
+// Don't declare arrow functions
+const App = () => (
+  <div>
+    <Logo />
+  </div>
+);
+
+// DO declare functions
+function App() {
+  return (
+    <div>
+      <Logo />
+    </div>
+  );
+}
+```
+
+###  1.7. <a name='Donameexports'></a>Do name exports
+
+It allows to export multiple values and it encourages the use of the same naming.
+
+```ts
+export function LoginPage() {
+  ...
+}
+```
+
+###  1.8. <a name='Doinlinetypeprops'></a>Do inline type props
+
+When using a type once, we can save some lines declaring the type inline.
```

Comment:
For me the key here is that it reduces the external dependencies of the function and therefore is more easy to move or change without errors.

Reply:
done!